### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by nandofw**
fixed display issue on command-line options page
reimplemented commit #288 
fixed network totals display issue #336 
fixed asset balance not updating properly
fixed dust issue on transfer asset output
fixed a mempool issue when create sub asset

### CODE REVIEW

File: src/addressindex.h [hunk 1]
```diff
@@ -68,7 +68,7 @@ struct CMempoolAddressDeltaKey {
     CMempoolAddressDeltaKey(int addressType, uint160 addressHash) {
         type = addressType;
         addressBytes = addressHash;
-        asset = "RTM";
```
Comment (npq7721) [inline src/addressindex.h:71]: was this added on other PR to fix getmempooladdresses calls?

Comment (nandofw) [reply src/addressindex.h:71]: yes

Comment (npq7721) [reply src/addressindex.h:71]: then it getting removed in this one and replace with asset,clear(). is it setting it somewhere else?

Comment (nandofw) [reply src/addressindex.h:71]: on this constructor the name should be empty, on the last pr i set it wrong and i only discover that it was wrong too late

**Review by npq7721 — APPROVED** [review]
LGTM

**Pull request merged**